notification-channel-config-service-impl/src/main/java/org/hypertrace/notification/config/service/NotificationChannelConfigServiceImpl.java

```diff
@@ -149,4 +151,19 @@ public void getNotificationChannel(
       responseObserver.onError(e);
     }
   }
+
+  private NotificationChannelMutableData getIdPopulatedData(NotificationChannelMutableData data) {
```

Comment:
Shouldn't the same be done in update too. That way, overtime if anyone updates existing configs, they also have the ID.

Reply:
This is being called from both update and create.